src/resources/ha-date-picker-style.html

```diff
@@ -0,0 +1,93 @@
+<dom-module id="ha-date-picker-vaadin-date-picker-styles" theme-for="vaadin-date-picker">
+  <template>
+    <style>
+      [part~="clear-button"] {
+        display: none;
+      }
+
+      [part~="toggle-button"] {
+        color: var(--disabled-text-color);
```

Comment:
I think you want to use secondary-text-color ?

Reply:
I copied that var from for example the drop-down list arrow. But that makes more sense.

Reply:
I also see that button needs to move a couple of px to the left...